mreg_cli/utilities/network.py

```diff
@@ -12,57 +12,12 @@
 from typing import Any
 
 from mreg_cli.api.models import Network
-from mreg_cli.log import cli_warning
+from mreg_cli.exceptions import EntityNotFound, EntityOwnershipMismatch, InputFailure
 from mreg_cli.types import IP_networkTV
 from mreg_cli.utilities.api import get, get_typed
 from mreg_cli.utilities.validators import is_valid_ip, is_valid_network
 
 
-def get_network_first_unused_ip(network: dict[str, Any]) -> str:
-    """Return the first unused ip from a given network.
-
-    Assumes network exists.
-    :param network: dict with network info (key: "network").
-    :return: Ip address string.
-    """
-    unused = get_network_first_unused(network["network"])
-    if not unused:
-        cli_warning("No free addresses remaining on network {}".format(network["network"]))
-    return unused
-
-
-def ip_in_mreg_net(ip: str) -> bool:
-    """Return true if the ip is in a MREG controlled network."""
-    ipt = ipaddress.ip_address(ip)
-    net = Network.get_by_ip(ipt)
-    return bool(net)
-
-
-def ipsort(ips: Iterable[Any]) -> list[Any]:
-    """Sort a list of ips."""
-    return sorted(ips, key=lambda i: ipaddress.ip_address(i))
-
-
-def ips_are_in_same_vlan(ips: list[str]) -> bool:
-    """Return True if all ips are in the same vlan."""
-    # IPs must be in a network, and that network must have a vlan for this to work.
-    last_vlan = ""
-    for ip in ips:
-        network = Network.get_by_ip(ipaddress.ip_address(ip))
-        if not network:
-            return False
-
-        if not network.vlan:
-            return False
-
-        if last_vlan and network.vlan != last_vlan:
-            return False
-
-        last_vlan = network.vlan
-
-    return True
-
-
 def get_network(ip: str) -> Network | None:
     """Return a network associated with given range or IP."""
```

Comment:
Is this function still used anywhere? I don't have access to an IDE to check it right now, but my gut feeling says it isn't — and if it is, it's probably a mistake.

Reply:
Culled in https://github.com/unioslo/mreg-cli/pull/255/commits/bf4ca3e81b1bca7f4081622d08edb998b35deab2